libbeat/_meta/fields.ecs.yml

```diff
@@ -0,0 +1,55 @@
+- key: ecs
```

Comment:
This file should probably be automatically generated from https://github.com/elastic/ecs/tree/master/schemas...

Reply:
Agree. Let's keep it manual for now to not add fields we don't use at the moment.